joomla/joomla-cms
# [com_redirect] Add search tooltip

#### Summary of Changes

Simple PR to add the search tooltip and element label (accessibility) in com_redirect.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Redirects, check the tooltip in the search field.
